From 35887c48dc49539edf5a3ae110a4a7b6aeee46f7 Mon Sep 17 00:00:00 2001
Subject: [PATCH 11/16] Add stubs for efidp_parse_* since they're in the
 headers.

These both set errno=ENOSYS and return -1 for now.

Signed-off-by: Peter Jones <pjones@redhat.com>
Signed-off-by: Curt Brune <curt@cumulusnetworks.com>

diff --git a/src/dp.c b/src/dp.c
index 8a167a1..1c25fd9 100644
--- a/src/dp.c
+++ b/src/dp.c
@@ -363,21 +363,21 @@ efidp_format_device_path(char *buf, size_t size, const_efidp dp, ssize_t limit)
 	return off+1;
 }
 
-#if 0
 ssize_t
 __attribute__((__visibility__ ("default")))
 efidp_parse_device_node(char *path, efidp out, size_t size)
 {
-
+	errno = -ENOSYS;
+	return -1;
 }
 
 ssize_t
 __attribute__((__visibility__ ("default")))
 efidp_parse_device_path(char *path, efidp out, size_t size)
 {
-
+	errno = -ENOSYS;
+	return -1;
 }
-#endif
 
 ssize_t
 __attribute__((__visibility__ ("default")))
